tools/sgen/input/FluentUIStyle.yml

```diff
@@ -445,6 +444,19 @@ AP_MSFListCellTokens:
   sublabelFont: $Typography.caption1
   trailingItemForegroundColor: $Colors.Foreground.neutral3
   trailingItemSize: $Icon.size.medium
+  
+AP_MSFHeaderFooterTokens:
+  backgroundColor: [ default: $Colors.Background.neutral1, divider: $Colors.Background.neutral5 ]
+  headerHeight: [ default: $Spacing.xxxlarge, divider: $Spacing.xlarge ]
+  horizontalHeaderPadding: $Spacing.medium
+  textColor: $Colors.Foreground.neutral3
+  textFont: $Typography.caption1
+  topHeaderPadding: $Spacing.xlarge
+
+MSFPrimaryHeaderTokens extends MSFHeaderFooterTokens:
+  textColor: $Colors.Foreground.neutral1
+  textFont: $Typography.headline
+  topHeaderPadding: $Spacing.medium
```

Comment:
Same for the padding here:

`padding: Insets($Spacing.medium, $Spacing.medium, $Spacing.medium, $Spacing.medium)`